src/Plugin/Field/FieldFormatter/CollaboraFileFormatterBase.php

```diff
@@ -24,6 +24,10 @@
 
 /**
  * Base class for Collabora Online field formatters.
+ *
+ * @template-extends EntityReferenceFormatterBase<\Drupal\file\FileInterface>
+ *
+ * @see ../../../../vendor/mglaman/phpstan-drupal/stubs/Drupal/Core/Field/Plugin/Field/FieldFormatter/EntityReferenceFormatterBase.stub
```

Comment:
TBD whether we want to keep this `@see` comment.\
For me it works nicely in phpstorm, the path becomes clickable.

Reply:
It's out of scope, let's drop it.

Reply:
The base class and the phpstan suppression was introduced with the other formatter.
I don't see it as out of scope to use this issue to clean up possible mistakes from the previous one.

Otherwise we need a new issue, I don't like it.

Reply:
Ok maybe we can indeed skip this change here, because we also do it on CoolPreview.